lib/public/views/Logs/ActiveColumns/index.js

```diff
@@ -15,7 +15,9 @@ const ACTIVE_COLUMNS = {
     title: {
         name: 'Title',
         visible: true,
-        size: 'cell-l',
+        expand: true,
+        expandTreshold: 40,
```

Comment:
This should either be calculated/dynamic or removed. This creates behavior such as the following (where more could/should have displayed but didn't):
![image](https://user-images.githubusercontent.com/25134477/85157204-743acc80-b25b-11ea-8682-a38556265fc3.png)
![image](https://user-images.githubusercontent.com/25134477/85157235-7ef56180-b25b-11ea-81df-286da86d933d.png)

Reply:
I cannot find another way to implement this with CSS/Javascript the treshhold is there to check for the text length to enable the plus/minus

Reply:
This is resolved in the latest version :)